praetorian_cli/handlers/run.py

```
@@ -0,0 +1,559 @@
+    click.echo(f'Command: {tool_name} {" ".join(args)}')
+    click.echo('─' * 60)
+
+    # Run with live output
+    proc = runner.run_streaming(args)
+    output_lines = []
+    try:
+        for line in proc.stdout:
+            click.echo(line, nl=False)
+            output_lines.append(line)
+        proc.wait(timeout=600)
+    except subprocess.TimeoutExpired:
```

Ruff (0.15.6)

[error] Undefined name `subprocess`

(F821)